### PR TITLE
develop: fix the bookshop example doc

### PR DESCRIPTION
**Pull request by Mini256**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-bookshop-schema-design.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ aliases: ['/zh/tidb/dev/bookshop-schema-design']
 
 Bookshop 是一个虚拟的在线书店应用，你可以在 Bookshop 当中便捷地购买到各种类别的书，也可以对你看过的书进行点评。
 
-## 导入数据
+为了方便你阅读本文档中的内容，我们将以 Bookshop 应用的[数据表结构](#数据表详解)和数据为基础来编写示例 SQL。
```
Comment (shichun-0415) [inline develop/dev-guide-bookshop-schema-design.md:11]:
```suggestion
为了让你更了解 Bookshop 的工作原理和使用方法，我们将以 Bookshop 应用的[数据表结构](#数据表详解)和数据为基础来编写示例 SQL。
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

File: develop/dev-guide-bookshop-schema-design.md (hunk 1 above)
Comment (Mini256) [inline develop/dev-guide-bookshop-schema-design.md:11]:
```suggestion
为了方便你阅读应用开发指南中的内容，我们将以 Bookshop 应用的[数据表结构](#数据表详解)和数据为基础来编写示例 SQL。本章节将为你介绍如何导入该应用的表结构和数据，以及其数据表结构的定义。
```

Comment (Mini256) [reply develop/dev-guide-bookshop-schema-design.md:11]: How about this? @shichun-0415

**Pull request merged**